poly-commitment/src/commitment.rs

```diff
@@ -674,16 +734,8 @@ where
 }
 
 impl<G: CommitmentCurve> SRS<G> {
-    /// This function verifies batch of batched polynomial commitment opening proofs
-    ///     batch: batch of batched polynomial commitment opening proofs
-    ///          vector of evaluation points
```

Comment:
so you're deleting this because now one can have info about the input types with the new docs?

Reply:
I removed because the comment seems outdated.